nidirect_gp/src/Entity/Gp.php

```diff
@@ -59,7 +59,7 @@
  *   },
  *   links = {
  *     "canonical" = "/gp/{gp}",
- *     "add-form" = "/gp/add",
+ *     "add-form" = "/node/add/gp",
```

Comment:
How does this work alongside GP Practice nodes? GPs are custom entities rather than node entities so I wouldn't expect that route to work

Reply:
It didn't seem to have any impact.